ProteGO/Common/DataModel/Bluetooth/BeaconId/ExpiringBeaconId.swift

```diff
@@ -1,16 +1,21 @@
 import Foundation
 
 struct ExpiringBeaconId {
+
+    let expirationDate: Date
```

Comment:
Powinno być `private`

Reply:
masz jakiś pomysł jak w takim razie sprawdzac ten parametr w testach?

Faktycznie, podczas działania aplikacji korzystanie z tego parametry bezpośrednio nie jest konieczne (wystarczy samo `isExpired`), ale nie miałem pomysłu jak inaczej to sprawdzić

Reply:
Też nie mam pomysłu na szybko, ok może być 👍